.github/workflows/config-change-0helm-pr.yaml

```diff
@@ -0,0 +1,72 @@
+name: config_change_alert
+
+on:
+  push:
+   branches: 
+     - staging
+   paths:
+      - docker.local/config/0chain.yaml
+      - docker.local/config/sc.yaml
+  
+  pull_request:
+    branches: 
+      - staging
+    types:
+      - closed
+    paths:
+      - docker.local/config/0chain.yaml
+      - docker.local/config/sc.yaml
+   
+jobs:
+  ConfigChange:
+    runs-on: docker-builds
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Get changed files using defaults
+        id: changed-files
+        uses: tj-actions/changed-files@v18.4
+
+      - name: "Clone 0helm"
+        if: ${{ contains(steps.changed-files.outputs.modified_files, 'docker.local/config/0chain.yaml') || contains(steps.changed-files.outputs.modified_files, 'docker.local/config/sc.yaml') }}
+        uses: actions/checkout@v2
+        with:
+          ref: one-pool-staging-test
+          repository: 0chain/0helm
+          fetch-depth: 1
+          path: ./0helm
+          token: ${{ secrets.ACCESS_TOKEN }}
+
+      - name: "Copy 0chain.yaml and sc.yaml in 0helm one-pool-staging-test branch"
+        if: ${{ contains(steps.changed-files.outputs.modified_files, 'docker.local/config/0chain.yaml') || contains(steps.changed-files.outputs.modified_files, 'docker.local/config/sc.yaml') }}
+        run: |
+          cp docker.local/config/0chain.yaml ../0helm/values/activeset/common/0chain.yaml
+          cp docker.local/config/sc.yaml ../0helm/values/activeset/common/sc.yaml
+
+      - name: Push Updated Code in 0helm one-pool-staging-test branch
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          personal_token: ${{ secrets.ACCESS_TOKEN }}
+          publish_dir: ./0helm/values/activeset/common
+          destination_dir: ./0helm/values/activeset/common
+          publish_branch: one-pool-staging-test
+          keep_files: true
+          external_repository: "0chain/0helm"
+          user_name: "github-actions[bot]"
+          user_email: "github-actions[bot]@users.noreply.github.com"
+
+  PR:
+    runs-on: 'docker-builds'
+    environment: 'HelmRepo'
+    needs: [ConfigChange]
+    steps:
+      - name: Raise PR
+        uses: devops-infra/action-pull-request@v0.5.5
+        with:
+          github_token: ${{ secrets.ACCESS_TOKEN }}
+          source_branch: one-pool-staging-test
+          target_branch: one-pool-staging
```

Comment:
raise the PR wrt to development branch.
Create new folder `updated configs` in 0helm repo on development branch. Add sc.yaml , 0chain.yaml, blobber.yaml, validator.yaml & 0box.yaml config file inside it and always raise the PR wrt to development branch.

Reply:
updated